app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,8 @@
     <%= yield %>
     </div>
     <%= render 'shared/footer' %>
+    <link href="https://cdn.jsdelivr.net/npm/select2@4.0.13/dist/css/select2.min.css" rel="stylesheet" />
```

Comment:
Probably just me being dumb, but doesn't yarn negate the need for a CDN? e.g. we dont download the stylesheet of bootstrap as a CDN in rails . Same question about jquery.